src/lib.rs

```diff
@@ -1280,7 +1280,7 @@ impl ExpressionStatus {
     }
 }
 
-mod errors {
+pub mod errors {
```

Comment:
Any chance it would work if you replaced `duct::errors::ErrorKind` with `duct::ErrorKind` and so on?